text/0064-asset-garbage-collection.md

```diff
@@ -0,0 +1,175 @@
+# Garbage Collection for Assets
+
+* **Original Author(s):**: @eladb, @kaizen3031593
+* **Tracking Issue**: #64
+* **API Bar Raiser**: @nlynch
+
+The asset garbage collection CLI will identify and/or delete orphaned CDK assets,
```

Comment:
I am not sure we should use the term "orphaned" - could be a sensitive term. Perhaps "dangling" or "unused" instead

Reply:
Good point. I think "unused" sounds best to my ears so I will replace with that